runtime/tests/interpreter/dynamic_casting_test.go

```diff
@@ -2334,6 +2336,15 @@ func testReferenceCastValid(t *testing.T, types, fromType, targetType string, op
 
 	switch operation {
 	case ast.OperationFailableCast:
+		if isResource {
+			AssertValuesEqual(
+				t,
+				inter,
+				interpreter.BoolValue(true),
```

Comment:
Master is still not merged to the stable-cadence branch yet. We can update it once it is merged. (although this test case might not break since go does a value-equality rather than instance equality for value types)